patroni/ha.py

```diff
@@ -1163,6 +1173,9 @@ def post_bootstrap(self):
         if not self.watchdog.activate():
             logger.error('Cancelling bootstrap because watchdog activation failed')
             self.cancel_initialization()
+        if not self.call_pre_promote(self.patroni.config):
```

Comment:
We can tell for sure that there is no former leader if we are doing the bootstrap.

Reply:
for sure, but does it mean we want to have a master that may be unfit as far as `pre_promote` is concerned ?